jina/hubble/hubio.py

```diff
@@ -787,7 +787,7 @@ def pull(self) -> str:
                     force=need_pull,
                 )
 
-                presented_id = getattr(executor, 'name', executor.uuid)
+                presented_id = getattr(executor, 'name') if getattr(executor, 'name') is not None else executor.uuid
```

Comment:
Does it make sense if we just use `executor.name` instead of `getattr(executor, 'name)`

Reply:
yes